job/status.go

```diff
@@ -0,0 +1,84 @@
+package job
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	k8upv1alpha1 "github.com/vshn/k8up/api/v1alpha1"
+)
+
+// SetConditionTrue tells the K8s controller at once that the status of the given Conditions is now "True"
+func (c *Config) SetConditionTrue(condition k8upv1alpha1.ConditionType, reason k8upv1alpha1.ConditionReason) {
+	c.patchConditions(metav1.ConditionTrue, reason, "", condition)
+}
+
+// SetConditionUnknownWithMessage tells the K8s controller at once that the status of the given Conditions is "Unknown"
+func (c *Config) SetConditionUnknownWithMessage(condition k8upv1alpha1.ConditionType, reason k8upv1alpha1.ConditionReason, message string, args ...interface{}) {
+	c.patchConditions(metav1.ConditionUnknown, reason, fmt.Sprintf(message, args...), condition)
+}
+
+// SetConditionFalse tells the K8s controller at once that the status of the given Condition is now "True" and
```

Comment:
CodeClimate says: `comment on exported method Config.SetConditionTrueWithMessage should be of the form "SetConditionTrueWithMessage ..."`